diff --git a/node_modules/typed-emitter/index.d.ts b/node_modules/typed-emitter/index.d.ts
index d18212c..8fd9a90 100644
--- a/node_modules/typed-emitter/index.d.ts
+++ b/node_modules/typed-emitter/index.d.ts
@@ -18,7 +18,7 @@ export type EventMap = {
  * myEmitter.emit("error", "x")  // <- Will catch this type error;
  * ```
  */
-interface TypedEventEmitter<Events extends EventMap> {
+export interface TypedEventEmitter<Events extends EventMap> {
   addListener<E extends keyof Events> (event: E, listener: Events[E]): this
   on<E extends keyof Events> (event: E, listener: Events[E]): this
   once<E extends keyof Events> (event: E, listener: Events[E]): this
